### ISSUE TITLE
Error while creating new vault

### ISSUE THREAD

**Issue by stepanselyuk** [jenkins-x/jx#5378]
### Summary 

I want to create new (and first) vault using `jx create vault` command on EKS.

### Steps to reproduce the behavior

```
jx create vault vault1 \
  --aws-auto-create=true \
  --aws-s3-region=us-west-2
```

### Expected behavior

CF stack for the vault created and executed, vault created in jenkins.

### Actual behavior

```
jx create vault vault1 \
  --aws-auto-create=true \
  --aws-s3-region=us-west-2
Region not specified, defaulting to us-west-2
Creating vault presiquite resources with following values, us-west-2, jenkins-x-domain, vault_us-west-2, vault-data
error: creating vault: creating vault: the server could not find the requested resource (post vaults.vault.banzaicloud.com)
```

AWS CF stack is created and executed. S3 bucket and DynamoDB table created.

### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 2.0.695
jenkins x platform 2.0.1119
Kubernetes cluster v1.13.10-eks-5ac0f1
kubectl            v1.15.3
helm client        v2.14.3+g0e7f3b6
helm server        v2.14.3+g0e7f3b6
git                2.23.0
Operating System   Mac OS X 10.14.6 build 18G95
```

### Jenkins type

- [ ] Serverless Jenkins X Pipelines (Tekton + Prow)
- [x] Classic Jenkins

### Kubernetes cluster

Created with eksctl.

### Operating system / Environment

Operating System:   Mac OS X 10.14.6 build 18G95

**Comment by hferentschik** [comment]
hi @stepanselyuk, thanks for the issue. I am wondering what you try to by running this command?

**Comment by stepanselyuk** [comment]
Pre-history is:

I've tried to install Jenkins X many times, every time with a fresh cluster. A few times when I specified `--helm-tls` it successfully created a vault and put there some secret data (cert-manager data).

Last time when I installed Jenkins X, it does not react on `--helm-tls` flag and run with just http.
Later I issued the command `jx upgrade ingress --cluster` and it installed cert-manager and generated certificates (from the third attempt). I discovered this time it did not go with creating a vault and placed secrets in k8s secrets. 

So later I just wanted to try to create a vault.
Even if it failed, I still see results on my attempt in k8s secrets (there were 3 secrets) :

```
vault1-auth-sa-token-tkz2m                 kubernetes.io/service-account-token   3      46m
vault1-token-vc6z9                         kubernetes.io/service-account-token   3      46m
```
I tried to remove them all (all three related to vault1), but the two above recreated automatically by some logic.
